src/firefly/js/metaConvert/ConverterFactory.js

```diff
@@ -57,7 +57,7 @@ export const converters = {
         hasRelatedBands : true,
         canGrid : true,
         maxPlots : 12,
-        makeRequest : makeLsstSdssPlotRequest,
+        makeRequest : makeLsstImagePlotRequest,
```

Comment:
You should not change the name here.  The right name is `makeLsstSdssPlotRequest` since we are showing SDSS data using LSST services. I don't think you need to touch this file.